config/config.go

```diff
@@ -119,21 +109,20 @@ type Metrics struct {
 }
 
 // OpenTelemetry collector configuration for tracing
-type OtelCollector struct {
+type TLSConfig struct {
 	CAName     string `yaml:"ca_name,omitempty"`
-	Protocol   string `yaml:"protocol,omitempty"` // http or https or grpc
 	SkipVerify bool   `yaml:"skip_verify,omitempty"`
-	TLSEnabled bool   `yaml:"tls_enabled,omitempty"`
 }
 
 // Tracing provides tracing configuration for the Kiali server.
 type Tracing struct {
-	CollectorType TracingCollectorType `yaml:"collector_type,omitempty"` // Possible values "otel" or "jaeger"
```

Comment:
I'm not opposed to removing this since everything in the foreseeable future will support otel but iirc there was a discussion (that I can no longer find) around keeping this in case in the future we again add more than one type. iirc @jmazzitelli wanted to keep the collector type for this reason?

Reply:
Right, I remember, now that you mention it... I think it might be in slack. Maybe we can wait to see what @jmazzitelli thinks about it?